.github/workflows/build.yml

```diff
@@ -141,13 +123,28 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Determine params
+        id: det-params
+        run: |
+          dep_name="${{ matrix.depends_name }}"
+          dep_opts="$(jq -r ".dep.include[] | select(.depends_name == \"${{ matrix.depends_name }}\") | .dep_opts" -c .github/workflows/matrix.json)"
+          dep_hash="$(echo -n ${dep_opts} | sha256sum | head -c 64)"
```

Comment:
only 64! What if there's a collision /s

Reply:
Oh, in that case, just use `sha256` instead of `sha256sum`

Reply:
Not a command available in the package.

```
dash@fc13e6631213:/src$ whereis sha256sum
sha256sum: /usr/bin/sha256sum
dash@fc13e6631213:/src$ whereis sha256
sha256:
```